apps/api-server/src/infrastructure/repositories/artistProfileRepository/index.test.ts

```
@@ -93,6 +94,22 @@ describe("artistProfileRepository", () => {
       expect(result?.isPublished()).toBe(true);
+      expect(result?.getPresentationPatternCode()).toBeNull();
+    });
+
+    it("表現パターンはマスタを leftJoin してコードで返す", async () => {
+      mock.enqueue(
+        [{ ...profileRow, presentationPatternCode: "editorial" }],
+        [],
+        [],
+        [],
+      );
+      const reader = createArtistProfileReader(mock.db as never);
```

ESLint

[error] `as` による型アサーションは使用しない。型ガード関数に切り出すか、正しく型付けされた契約から値を取得する。

(no-restricted-syntax)